release_notes.md

```diff
@@ -118,7 +118,7 @@
 
 ## User impact and migration instructions
 
-1. [I] The "metadataOnly" query parameter will now always be set to False in the get_X_from_data_provider methods. It’s
+1. [I] The "metadataOnly" query parameter will now always be set to False in the get_X_from_data_provider methods. Its
```

Comment:
??

Reply:
Must be `pre-commit`, I havent touched it 😅